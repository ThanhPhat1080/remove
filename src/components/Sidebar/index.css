.sidebar-wrapper {
  width: 200px;
  height: 100%;
  background: var(--dark-charcoal);
  padding: 30px 0px;
  position: fixed;
}
