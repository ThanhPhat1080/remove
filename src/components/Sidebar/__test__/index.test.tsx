// Lib
import { render, screen } from '@testing-library/react';

// Components
import Sidebar, { SidebarProps } from '@components/Sidebar';

// Mocks
import { mockDataUser } from '@mocks/mockDataUser';

describe('Test Sidebar component', () => {
  const initSidebar: SidebarProps = {
    // @ts-ignore
    users: mockDataUser,
  };

  test('It should match DOM Snapshot', () => {
    const { container } = render(<Sidebar {...initSidebar} />);

    expect(container).toMatchSnapshot();
  });

  test('It should match data for Sidebar component', () => {
    render(<Sidebar {...initSidebar} />);

    expect(screen.getAllByRole('img')[0].getAttribute('src')).toEqual(
      'https://haycafe.vn/wp-content/uploads/2022/03/Avatar-hai-1.jpg',
    );

    expect(screen.getAllByRole('link')[0].textContent).toEqual('user1');
  });
});
