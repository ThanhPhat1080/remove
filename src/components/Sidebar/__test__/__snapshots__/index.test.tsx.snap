// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Sidebar component It should match DOM Snapshot 1`] = `
<div>
  <aside
    class="sidebar-wrapper"
  />
</div>
`;
